### DISCUSSION TITLE
如何更改未登录评论者的头像 || How to change the avatar of commenters who have not logged in

### DISCUSSION THREAD

**Discussion by 2176608** [walinejs/waline#2025] [Q&A] [answered]
### Describe the bug | 问题描述

更改未登录评论者的头像
![Snipaste_2023-07-15_20-03-56](https://github.com/walinejs/waline/assets/90504717/acde8573-2415-4541-9284-c470968daadf)

### Website URL | 问题网站

2176608.github.io

### Where your waline deploy? | 服务部署在哪里？

Deta

### Where your comment data store? | 数据存储在哪里？

GitHub

---
### Describe the bug | Problem description

Change the avatar of a commenter who is not logged in
![Snipaste_2023-07-15_20-03-56](https://github.com/walinejs/waline/assets/90504717/acde8573-2415-4541-9284-c470968daadf)

### Website URL | Problem Website

2176608.github.io

### Where your waline deploy? | Where is the service deployed?

Deta

### Where your comment data store? | Where is your comment data store?

GitHub

**Comment by lizheming** [comment, marked as answer]
未登录用户的头像默认展示的是 Gravatar 的头像，没有的话会有一个默认头像。这个是可以通过 GRAVATAR_STR 配置的。
https://wiki.libravatar.org/api/
https://waline.js.org/reference/server/env.html#%E6%98%BE%E7%A4%BA

**Reply by 2176608** [reply]
感谢！

---
grateful!
